go/libkb/sig_chain.go

```diff
@@ -514,6 +516,15 @@ func (sc *SigChain) Store(m MetaContext) (err error) {
 	return nil
 }
 
+func (sc *SigChain) checkUnstubs(low int, unstubs map[keybase1.Seqno]LinkID) error {
+	for _, link := range sc.chainLinks[low:] {
```

Comment:
sanity check all of the unstubs are found?

Reply:
I was thinking about that...sure....